Fixes #2375

SalmonMode opened the pull request:
Fixes #2375 

I noticed this because my install of PHP was referencing a config file in `/etc/php/7.2/cli/`, instead of `/etc/php/7.0/apache2/`.

connortechnology commented on docs/installationguide/debian.rst:
```diff
@@ -108,6 +108,21 @@ required apache modules.
+
+::
+
+    php --ini | grep 'Configuration File'
```
The problem is that this will give you the path to the config file used when running from the command line, not the one that is in use under apache.

We need to look in /etc/apache2/mods-enabled to see which php module is loaded.  On mine we have php7.2.conf and php7.2.load so we our php config is /etc/php/7.2/apache2/php.ini

SalmonMode replied:
Ah, interesting. I'm not actually super familiar with PHP and apache. This actually clarified a lot of things for me, thanks!

So if I understand correctly, one should always be able to expect the PHP config file for apache2 to be in an `apache2` folder inside `/etc/php/<version>/` or `/etc/php5/` (if version 5). So you could determine the version by checking `/etc/apache2/mods-enabled`, and then piece together where the config file should be, correct?

connortechnology replied:
This php --ini still gives you the path to the config file used when running php from the command line. NOT the one that is used by apache.

SalmonMode replied:
Sorry, this file was supposed to be reverted to what it was originally. I think I see how I missed this. I'll make sure it gets reverted this time

SalmonMode commented:
Changed per the convo in #2375. I just added a duplicate of the 16.04 guide and changed the Ubuntu version and where the PHP config file could be found.

SalmonMode commented:
Ok, definitely fixed that issue with the debian instructions. It should be all set now. Sorry about that :sweat_smile:
